### PR TITLE
Test model

### PR DESCRIPTION
**Pull request by BaranovMykola**
Implemented AnswerType enum, Answer, Question, Ticket, AnswerChecker classes.

### CODE REVIEW

File: TestRuleModel/Test/AnswerChecker.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    public static class AnswerChecker
+    {
+        public static bool CheckAnswer(IEnumerable<Answer> ans)
+        {
+            return CheckAnswer(ans, AnswerType.True);
+        }
+
+        public static bool CheckAnswer(IEnumerable<Answer> ans, AnswerType type)
+        {
+            return ans.All(s => s.Type == type);
```
Comment (OlyaOlga) [inline TestRuleModel/Test/AnswerChecker.cs]: How this method will check questions with multiple choice?

File: TestRuleModel/Test/AnswerChecker.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
+﻿namespace TestRuleModel.Test
+{
+    using System.Collections.Generic;
+    using System.Linq;
+
+    public static class AnswerChecker
+    {
+        public static bool CheckAnswer(IEnumerable<Answer> ans)
```
Comment (OlyaOlga) [inline TestRuleModel/Test/AnswerChecker.cs]: Why do you need function reloading here?

File: TestRuleModel/Test/Question.cs [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+    public class Question
+    {
+        public Question()
+        {
+        }
+
+        public Question(string qustionText, string imagePath, IEnumerable<Answer> choiseList)
+        {
+            QustionText = qustionText;
```
Comment (OlyaOlga) [inline TestRuleModel/Test/Question.cs]: question

File: TestRuleModel/Test/Question.cs [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+            ImagePath = imagePath;
+            ChoiseList = choiseList;
+        }
+
+        public string QustionText { get; set; }
```
Comment (OlyaOlga) [inline TestRuleModel/Test/Question.cs]: :)

**Comment by BaranovMykola** [comment]
Fixed code due your warnings

**Comment by BaranovMykola** [comment]
Improved AnswerChecker class logic

**Pull request merged**